ReactNativeClient/lib/joplin-database.js

```diff
@@ -7,6 +7,7 @@ const structureSql = `
 CREATE TABLE folders (
 	id TEXT PRIMARY KEY,
 	title TEXT NOT NULL DEFAULT "",
+	icon TEXT NOT NULL DEFAULT "",
```

Comment:
To modify the database, you need to create a migration, like is done for example there: https://github.com/laurent22/joplin/blob/master/ReactNativeClient/lib/joplin-database.js#L649

If you have any question on how to make it work let me know.

Reply:
I'm starting to wrap my head around this a bit more but databases are something i barely know a thing about, and I'll willingly take help with it. I would like to learn how that part of the code works but also know that much of that is something I'll need to dedicate quite a bit of time to on my own time.

Reply:
Target version 27 is the only one that deals with a file and the final product is always going to be a new database file, so am I correct for assuming that everytime a new property is added to the database, the highest target version is always incremented by 1?